public/app/features/dashboard/components/ShareModal/ShareLink.tsx

```diff
@@ -137,13 +137,11 @@ export class ShareLink extends PureComponent<Props, State> {
         {panel && config.rendererAvailable && (
           <>
             {isDashboardSaved && (
-              <div className="gf-form">
-                <a href={imageUrl} target="_blank" rel="noreferrer" aria-label={selectors.linkToRenderedImage}>
-                  <Icon name="camera" />
-                  &nbsp;
-                  <Trans i18nKey="share-modal.link.rendered-image">Direct link rendered image</Trans>
-                </a>
-              </div>
+              <a href={imageUrl} target="_blank" rel="noreferrer" aria-label={selectors.linkToRenderedImage}>
+                <Icon name="camera" />
+                &nbsp;
+                <Trans i18nKey="share-modal.link.rendered-image">Direct link rendered image</Trans>
+              </a>
```

Comment:
```suggestion
                <TextLink href={imageUrl} external icon={'camera'} aria-label={selectors.linkToRenderedImage}>
                  <Trans i18nKey="share-modal.link.rendered-image">Direct link rendered image</Trans>
                </TextLink>
```
This should probably be a `TextLink`, but it needs to be modified to accept the translation components as children. 
Or this could work too I guess: 
```jsx
<TextLink href={imageUrl} external icon={'camera'} aria-label={selectors.linkToRenderedImage}>
    {t('share-modal.link.rendered-image', 'Direct link rendered image')}
</TextLink>
```